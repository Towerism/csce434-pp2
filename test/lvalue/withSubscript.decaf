int asdf() {
    this[asdf = 5];
}