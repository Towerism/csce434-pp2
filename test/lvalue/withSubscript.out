
   Program: 
  1   FnDecl: 
         (return type) Type: int
  1      Identifier: asdf
         (body) StmtBlock: 
  2         ArrayAccess: 
  2            This: 
  2            (subscript) AssignExpr: 
  2               FieldAccess: 
  2                  Identifier: asdf
  2               Operator: =
  2               IntConstant: 5
